sphinx_needs/directives/needimport.py

```diff
@@ -115,21 +112,21 @@ def run(self) -> Sequence[nodes.Node]:
                     f"Could not load needs import file {correct_need_import_path}"
                 )
 
-            errors = check_needs_file(correct_need_import_path)
+            try:
+                with open(correct_need_import_path) as needs_file:
+                    needs_import_list = json.load(needs_file)
+            except json.JSONDecodeError as e:
```

Comment:
`OSError` should also be caught, but there is already a TODO

Reply:
done

Reply:
(knowing what exceptions to handle is always a pain, I was reading this with interest recently https://discuss.python.org/t/extend-type-hints-to-cover-exceptions/23788/51)